llvm/test/Transforms/LoopVectorize/AArch64/induction-costs-sve.ll

```diff
@@ -522,31 +522,31 @@ define void @trunc_ivs_and_store(i32 %x, ptr %dst, i64 %N) #0 {
 ; PRED:       pred.store.continue:
 ; PRED-NEXT:    [[TMP23:%.*]] = extractelement <4 x i1> [[ACTIVE_LANE_MASK]], i32 1
 ; PRED-NEXT:    br i1 [[TMP23]], label [[PRED_STORE_IF3:%.*]], label [[PRED_STORE_CONTINUE4:%.*]]
-; PRED:       pred.store.if3:
+; PRED:       pred.store.if2:
```

Comment:
Is there a way to remove or reduce the amount of such irrelevant naming changes?

Reply:
Trying to avoid additional redundant changes.

Reply:
Unfortunately I don't think so, as the numbering depends on the exact order in which names are created that need de-duplication (there's a single counter for all names)

Reply:
OK. Just curious - are there now fewer(?) de-duplications taking place, or only a reordering of de-duplications?

Reply:
Tried to strop as many unrelated changes from the diffs, but also added some new ones in the ones with predicated blocks to update the block variables.